### PR TITLE
♻️ Rewrite ampproject.org links in examples/src to internal ones

### PR DESCRIPTION
**Pull request by robinvanopstal**

### CODE REVIEW

File: examples/source/0.introduction/Banner_Ad.html [hunk 1]
```diff
@@ -45,9 +45,9 @@
   </a>
 
   <!--
-   We include a tracking pixel to record that the ad was rendered using [amp-pixel](https://www.ampproject.org/docs/reference/components/amp-pixel).
+   We include a tracking pixel to record that the ad was rendered using [amp-pixel]({{g.doc('/content/amp-dev/documentation/components/reference/amp-pixel.md', locale=doc.locale).url.path}}).
 
-  <p>Alternatively one could make use of [amp-analytics](https://www.ampproject.org/docs/reference/extended/amp-analytics.html) to enable much more comprenhensive measure activity.</p>
+  <p>Alternatively one could make use of [amp-analytics]({{g.doc('/content/amp-dev/documentation/components/reference/amp-analytics.md', locale=doc.locale).url.path}}) to enable much more comprenhensive measure activity.</p>
```
Comment (sebastianbenz) [inline examples/source/0.introduction/Banner_Ad.html]: `components/reference` has changed to `components`

**Comment by sebastianbenz** [comment]
What's the status here?

File: examples/source/0.introduction/Banner_Ad.html (hunk 1 above)
Comment (keevkeev) [reply examples/source/0.introduction/Banner_Ad.html]: the file path here is correct, only the external, user facing links changed

**Comment by keevkeev** [comment]
> What's the status here?

hey @sebastianbenz, thanks for your patience on this one. I'm currently working through all the examples and fixing broken links (eg. nearly all links on https://amp.dev/documentation/examples/e-commerce/amp_for_e-commerce_getting_started/?format=websites lead to a 404). I hope to get this resolved ASAP

**Pull request merged**